README.md

```
@@ -0,0 +1,23 @@
+# telegram-member-scraper
+Export all users from member groups using Telethon. 
+
+This is a Python script based off a tutorial for scraping Telegram users from groups. 
+The original code allowed the user to export an individual group's members into one CSV file. 
+I have now adapted the code so that each group's members can be outputed into individual CSV files which are
+appropriately named as the corresponding groups.
+
+## Requirements:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)